app/views/users/passwords/edit.html.erb

```diff
@@ -1,4 +1,5 @@
-<h2>Change your password</h2>
+<% @pageTitle = "Set new Password | CAT Forum" %>
+<% @PageDescription = "Set new password for your account" %><h2>Change your password</h2>
 
```

Comment:
please see beside  `<% @PageDescription = "Set new password for your account" %>` in line 2.the <h2> tag is after the erb tag.